docs/docs-content/clusters/public-cloud/aws/eks-hybrid-nodes/create-hybrid-node-pools.md

```diff
@@ -0,0 +1,368 @@
+
+   ```shell
+       podCIDRs:
+       - 192.168.5.0/25
+   ```
+
+3. Access your VPN configuration interface. This could be a web UI, command-line tool, or API, depending on your VPN
+   solution.
+
+4. For each hybrid node, add the following entries.
+
+   | **Field**          | **Description**                                                                                                         | **Example**        |
+   | ------------------ | ----------------------------------------------------------------------------------------------------------------------- | ------------------ |
+   | Destination        | Use the `podCIDRs` value for the hybrid node discovered in step 2.                                                      | `192.168.4.128/25` |
+   | Next Hop / Gateway | Specify the IP address of the hybrid node as listed in the CiliumNode resource under `internalIP` discovered in step 1. | `192.168.5.101`    |
+
+5. Ensure the routes are saved and applied. The process varies depending on the VPN solution.
+
+### Validate
+
+1. From a pod in your Amazon EKS cluster, attempt to reach an active pod on a hybrid node.
+
+   Replace `<podName>` with a pod in your Amazon EKS cluster and `<hybridPodIp>` with an IP address from an active pod
+   on a hybrid node.
+
+   ```shell
+   kubectl exec --interactive --tty <podName> -- ping <hybridPodIp>
+   ```
+
+2. Check that the ping statistics from the output show a healthy connection.
+
+   Example healthy output.
+
+   ```shell
+   PING 192.168.5.10 (192.168.5.10): 56 data bytes
+   64 bytes from 192.168.5.10: icmp_seq=1 ttl=63 time=28.382 ms
+   64 bytes from 192.168.5.10: icmp_seq=2 ttl=63 time=27.359 ms
+   64 bytes from 192.168.5.10: icmp_seq=3 ttl=63 time=29.412 ms
+   64 bytes from 192.168.5.10: icmp_seq=4 ttl=63 time=30.345 ms
+
+   --- 192.168.5.10 ping statistics ---
+   4 packets transmitted, 4 packets received, 0% packet loss
+   round-trip min/avg/max/stddev = 27.359/28.875/30.345/1.091 ms
+   ```
+
+## When to Manually Repave Hybrid Node Pools
+
+Your hybrid node pools require manual repaving in these scenarios:
+
+- After modifying the **Access Management** settings of your Amazon EKS cluster in Palette. Refer to steps 11 through 13
+  in [Import Cluster](./import-eks-cluster-enable-hybrid-mode.md#import-cluster).
+- After changing an edge host's **VPN Server IP**. Refer to step 7 in [Create Node Pool](#create-node-pool).
+
+These changes do not take effect until you repave the affected node pools.
+
+- For **Access Management** changes, repave all hybrid node pools.
+- For **VPN Server IP** changes, repave only the node pool containing the modified edge host.
+
+### Prerequisites
+
+- Your Palette account role must have the `cluster.update` permission to edit clusters. Refer to the
+  [Cluster Profile](../../../../user-management/palette-rbac/project-scope-roles-permissions.md#cluster-profile)
+  permissions for guidance.
+
+### Trigger Repave on Hybrid Node Pool
```

Comment:
This section looks good, but manual testing is required to confirm that this procedure actually works for both configuration updates.

Reply:
They will be once I finish my own dev / dev testing.

Right now it's looking like I'll have to implement proper repaves when those fields are edited within the 4.5.c timeframe. Because the suggested workaround is not actually working.